lib/analysis/ProjectAnalysis.ts

```diff
@@ -18,6 +18,7 @@ import {
     BaseParameter,
     RemoteRepoRef,
 } from "@atomist/automation-client";
+import { Fingerprint } from "@atomist/automation-client/lib/project/fingerprint/Fingerprint";
```

Comment:
Just a note for next week but we plan to move Fingerprint into custom ingestion so there will be an AtomistFingerprint in types.ts.  Compatible with this signature though.